playbooks/roles/airship-setup-deployer/defaults/main.yml

```diff
@@ -1,9 +1,19 @@
 ---
 # defaults file for suse-airship-deploy
 
+# IP on the Data Center Management (DCM) network
+dcm_vip: null
+
+# IP on the external network
+ext_vip: null
```

Comment:
do we need to add these? isnt that covered by the change above?

Reply:
It is covered. But having it here is imo good to have the available variables documented.

Reply:
@Itxaka is that answer ok? Or do you want me to change something here?